### PR TITLE
Register container changes

### PR DESCRIPTION
**Pull request by mightyiam**

### CODE REVIEW

File: src/test/unit/specific_feature_tests/set-root-container.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        // h1, h2, etc.
+        loadDefaultPlugins: false
+    });
+}});
+
+// We test that each tag (as in element) can be changed to each tag.
+// This creates this array of tags.
+var pairs = [];
+// what about the div?
+['p', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'pre']
```
Comment (mightyiam) [inline src/test/unit/specific_feature_tests/set-root-container.js]: What about when the default root container is a `div`?

**Comment by mightyiam** [comment]
The tests for `setRootContainer` may be incomplete but they're more than the zero tests we had until now. Please review.

File: src/test/unit/utils.js [hunk 2]
```diff
@@ -134,10 +140,12 @@ function prepareUnitTestModule(args) {
     }
 
     args.options.postInit = function (wym) {
-        // TODO: We should not load all these plugins by default.
-        wym.listPlugin = new ListPlugin({}, wym);
-        wym.tableEditor = wym.table();
-        wym.structuredHeadings();
+        if (args.loadDefaultPlugins) {
+            // TODO: We should not load all these plugins by default.
```
Comment (winhamwr) [inline src/test/unit/utils.js]: I'd say that the listPlugin and tableEditor should be loaded in >90% of use cases and we need to make sure they're test. In a perfect world, we would have tests both with them and without them, but that seems like a lot of extra tests.

`structuredHeadings` is a bit of a special case, at least right now.

**Comment by winhamwr** [comment]
> incomplete but they're more than the zero tests we had until now

Agreed.

**Comment by mightyiam** [comment]
Awesome! Thank you!

**Pull request merged**